src/controller/user-controller.js

```diff
@@ -0,0 +1,51 @@
+const client = require('../db');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const jwtTokens = require('../utils/jwt-helpers');
+
+
+
+
+const getAllUserHandler = async (req, res) => {
+    try {
+        client.query(`Select * from users`, (err, result)=>{
```

Comment:
this is okay but what would make sense is to abstract your query into their own file. so have like a query folder then in each you can have userQuery and add all this, i don't think you need to be doing client.end